src/pages/History/HistoryVideoCard/HistoryVideoCard.jsx

```diff
@@ -0,0 +1,39 @@
+import "./HistoryVideoCard.css";
+import { Link } from "react-router-dom";
+import { requestDeleteVideoInHistory } from "../../../utils/server-request";
+import { useAuth } from "../../../context/auth-context";
+
+const HistoryVideoCard = ({ video }) => {
+  const {
+    userState: { userToken },
+    userDispatch,
+  } = useAuth();
+
+  const headers = { headers: { authorization: userToken } };
```

Comment:
Does not have to be initialized here, can instead be directly set in the API call itself

Reply:
did this to make code more clean and readable